packages/app-builder/src/components/hooks/use-introspection/types.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  IntrospectionObjectType,
+  IntrospectionOutputTypeRef,
+  IntrospectionNonNullTypeRef,
+  IntrospectionListTypeRef,
+  IntrospectionField,
+} from 'graphql'
+
+export const OBJECT = 'OBJECT'
+export const NON_NULL = 'NON_NULL'
+export const LIST = 'LIST'
+export const SCALAR = 'SCALAR'
```

Comment:
Could use an enum here? That way we can group them with a generic type of say `ObjectType` where `ObjectType` is one of these values

Reply:
Comes in handy for types like `{[keyof ObjectType]: Function}`